src/main/java/org/opensearch/security/dlic/rest/api/PatchableResourceApiAction.java

```diff
@@ -172,6 +172,10 @@ public void onResponse(IndexResponse response) {
             });
     }
 
+    protected boolean verifyPermissionFor(final JsonNode jsonNode, final Method method) {
```

Comment:
Ahh this is interesting, all endpoints except `NodesDN`, `Allowlist` and SSL certs before any action will be applied verify is the current user is super admin or not, while others verify what the current user wants to change if it is `hidden` or `reserved` fields the action is forbidden until the use is a super admin one. IMHO Action Handlers need to be clean since in some case you can see such code:
https://github.com/opensearch-project/security/blob/b86eba2ae57dc99a82d8b45cfbcefda9aecafb2e/src/main/java/org/opensearch/security/dlic/rest/api/PatchableResourceApiAction.java#L233 
which should be in `ActionGroupsActionHanlder`.

Reply:
I mean that the default behave is that you can save, update and delete most of the configuration objects .

Reply:
Thank you for the explanation.